server/plugin.go

```diff
@@ -213,18 +235,76 @@ func (p *Plugin) startMeeting(user *model.User, channel *model.Channel, meetingI
 
 	meetingUntil := ""
 	if JWTMeeting {
-		meetingUntil = "Meeting link valid until: " + meetingLinkValidUntil.Format("Mon Jan 2 15:04:05 -0700 MST 2006")
+		meetingUntil = l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID:    "jitsi.start_meeting.meeting_link_valid_until",
+				Other: "Meeting link valid until: {{.Datetime}}",
+			},
+			TemplateData: map[string]string{"Datetime": meetingLinkValidUntil.Format("Mon Jan 2 15:04:05 -0700 MST 2006")},
+		})
 	}
 
-	meetingTypeString := "Meeting ID"
+	meetingTypeString := l.MustLocalize(&i18n.LocalizeConfig{
+		DefaultMessage: &i18n.Message{
+			ID:    "jitsi.start_meeting.meeting_id",
+			Other: "Meeting ID",
+		},
+	})
 	if meetingPersonal {
-		meetingTypeString = "Personal Meeting ID (PMI)"
+		meetingTypeString = l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID:    "jitsi.start_meeting.personal_meeting_id",
+				Other: "Personal Meeting ID (PMI)",
+			},
+		})
+	}
+
+	slackMeetingTopic := meetingTopic
+	if slackMeetingTopic == "" {
+		slackMeetingTopic = l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID:    "jitsi.start_meeting.default_meeting_topic",
+				Other: "Jitsi Meeting",
+			},
+		})
 	}
 
 	slackAttachment := model.SlackAttachment{
-		Fallback: fmt.Sprintf("Video Meeting started at [%s](%s).\n\n[Join Meeting](%s)\n\n%s", meetingID, meetingURL, meetingURL, meetingUntil),
-		Title:    meetingTopic,
-		Text:     fmt.Sprintf("%s: [%s](%s)\n\n[Join Meeting](%s)\n\n%s", meetingTypeString, meetingID, meetingURL, meetingURL, meetingUntil),
+		Fallback: l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID: "jitsi.start_meeting.fallback_text",
+				Other: `Video Meeting started at [{{.MeetingID}}]({{.MeetingURL}}).
+
+[Join Meeting]({{.MeetingURL}})
+
+{{.MeetingUntil}}`,
```

Comment:
Given that this line is the output of a translation, can it just the append without being part of the translation message?

Reply:
Yes, for this specific case, I think makes sense, because is not context related.

Reply:
Done